### PR TITLE
Devo enh 3 - python-sdk upgrade.

### PR DESCRIPTION
**Pull request by manas-metron**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Devo/Integrations/Devo_v2/Devo_v2.py [hunk 1]
```diff
@@ -155,25 +160,59 @@ def alert_to_incident(alert, user_prefix):
...
+
+    # Query the data
+    response = self.query(
+        query=linq_query,
+        dates={'from': timestamp_to_date(start), 'to': timestamp_to_date(stop)}
     )
 
     try:
         data = json.loads(response)
-        error_checking.check_status(data)
```
Comment (sahil-metron) [inline Packs/Devo/Integrations/Devo_v2/Devo_v2.py:172]: How are we handling this error_checking related stuff ?

File: Packs/Devo/Integrations/Devo_v2/Devo_v2.py [hunk 2]
```diff
@@ -763,16 +808,20 @@ def multi_table_query_command(offset, items):
...
-        end_point=READER_ENDPOINT,
-        verify=not ALLOW_INSECURE,
-        timeout=query_timeout,
-    )
-    ds_read.get_types = partial(get_types, ds_read)
+    api = Client(auth={"token": READER_OAUTH_TOKEN},
+                 address=READER_ENDPOINT,
+                 config=ClientConfig(response="json", stream=False))
+
+    # ds_read = ds.Reader(
```
Comment (sahil-metron) [inline Packs/Devo/Integrations/Devo_v2/Devo_v2.py]: Remove these comments

**Review by sahil-metron — CHANGES_REQUESTED** [review]

File: Packs/Devo/Integrations/Devo_v2/Devo_v2.py (hunk 1 above)
Comment (manas-metron) [reply Packs/Devo/Integrations/Devo_v2/Devo_v2.py:172]: I've added response validator.

File: Packs/Devo/Integrations/Devo_v2/Devo_v2.py (hunk 2 above)
Comment (manas-metron) [reply Packs/Devo/Integrations/Devo_v2/Devo_v2.py]: yes

**Review by sahil-metron — APPROVED** [review]

**Pull request closed without merging**